README.md

```diff
@@ -312,23 +323,23 @@ taq deploy nft.tz -e "testing"
 └──────────┴──────────────────────────────────────┴───────┴──────────────────┴────────────────────────────────┘
 ```
 
-:tada: Hooray ! We have finished the backend :tada:
+**:tada: Hooray! We have finished the backend :tada:**
 
 # :performing_arts: NFT Marketplace front
 
 ## Get the react boilerplate
 
-To win time, we have a boilerplate ready for the UI here : [https://github.com/marigold-dev/training-nft-1/tree/main/reactboilerplateapp](https://github.com/marigold-dev/training-nft-1/tree/main/reactboilerplateapp)
+To win time, we have a [boilerplate ready for the UI](https://github.com/marigold-dev/training-nft-1/tree/main/reactboilerplateapp)
 
-Copy this code into your folder (:warning: assuming you have cloned this repo, and your current path is $REPO/training)
+Copy this code into your folder (:warning: assuming you have cloned this repo, and your current path is `$REPO/training`)
 
 ```bash
 cp -r ../reactboilerplateapp/ ./app
 ```
 
-> Note : if you want to understand how it has been made from scratch look at this training => [https://github.com/marigold-dev/training-dapp-1#construction_worker-dapp](https://github.com/marigold-dev/training-dapp-1#construction_worker-dapp)
+> Note : if you want to understand how it has been made from scratch look at [this training](https://github.com/marigold-dev/training-dapp-1#construction_worker-dapp)
 
-Generate the typescript classes from Michelson code to your frontend app and run the server
+Generate the Typescript classes from Michelson code to your frontend app and run the server
 
 ```bash
 taq install @taqueria/plugin-contract-types
```

Comment:
Please briefly describe what each line is doing. I don't copy/paste/execute command if I don't know what they are supposed to do.